lib/cdo/github.rb

```diff
@@ -0,0 +1,66 @@
+require 'cdo/chat_client'
+require 'net/http'
+
+# This module serves as a wrapper around the GitHub API, offering convenience
+# methods for pull request functionality
```

Comment:
Why not [Octokit](https://github.com/octokit/octokit.rb)?  We already use it in [`create-release`](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/create-release).